File: 001/001.c
#include <stdio.h>

int total = 0;

int main() {

  for (int i=1; i<1000; i++) {
    if ( !(i%3) || !(i%5) ){
      total += i;
    }
  }
   printf("%i\n", total);
   return 0;
}

File: 001/001.rs
fn main(){
    let mut total = 0;

    for i in 1..1000{
        if i%3==0{
            total += i;
        } else if i % 5 == 0{
            total += i;
        }
    }
    
    println!("total is {}", total);
}